### PR TITLE
(WIP) Default tags

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

File: api/models/Community.js [hunk 1]
```diff
@@ -90,10 +90,26 @@ module.exports = bookshelf.Model.extend({
       return newPost.save({created_at: time, updated_at: time}, {transacting: trx})
       .then(() => Promise.all(_.flatten([
         self.posts().attach(newPost, {transacting: trx}),
+        Tag.updateForPost(newPost, newPost.get('type'), trx),
```
Comment (robbiecarlton) [inline api/models/Community.js]: This line causes the method to break with an error about the transaction,

```
error: current transaction is aborted, commands ignored until end of transaction block
at [object Object].Connection.parseE (/Users/robbiecarlton/work/hylo/hylo-node/node_modules/pg/lib/connection.js:539:11)
```

File: test/unit/models/Membership.test.js [hunk 2]
```diff
@@ -1,53 +1,72 @@
...
-
-    it('returns nothing for a blank user id', function() {
-      return Membership.find(null, community.id).then(function(membership) {
-        expect(membership).not.to.exist;
-      });
-    });
-
-    it('does not return an inactive membership', function() {
+          user.save(),
+          Tag.createDefaultTags()
```
Comment (levity) [inline test/unit/models/Membership.test.js]: i think this is supposed to go in the `before` block for the `create` tests, not the `find` tests

File: api/models/Community.js (hunk 1 above)
Comment (levity) [inline api/models/Community.js]: not totally sure it's the root cause, but you have a few finds and loads downstream that don't use the transaction

**Comment by robbiecarlton** [comment]
@levity adding the missing transaction fixed it, 
and I moved the default tag creation in that test.
Good to merge?

**Comment by levity** [comment]
sweet. I wish the database error were a bit more descriptive. I guess the behavior is that if you make a database query without the transaction parameter, it closes the transaction first. anyway :shipit:

**Pull request merged**